### ISSUE TITLE
How can I find out if that model is overfitting on CropGBM?

### ISSUE THREAD

**Issue by jeong2624** [YuetongXU/CropGBM#3]
Even if the cross-validation results indicate a high score, it is essential to assess whether the model is overfitting. 
Therefore, how can I determine if the model exhibits overfitting based on the cross-validation results?

**Comment by YuetongXU** [comment]
Cross-validation is generally used as an initial indicator to assess whether a model is overfitting. For further judgment, it is better to conduct secondary validation using an independent dataset. For example, when we use the CUBIC population data of maize as the training set, we can use the AMP population data of maize as the validation set.
